processor/src/main/java/com/joom/lightsaber/processor/LightsaberParameters.kt

```diff
@@ -17,12 +17,33 @@
 package com.joom.lightsaber.processor
 
 import java.io.File
+import java.nio.file.Path
 
 data class LightsaberParameters(
-  val inputs: List<File>,
-  val outputs: List<File>,
-  val classpath: List<File>,
-  val bootClasspath: List<File>,
-  val gen: File,
-  val projectName: String
-)
+  val inputs: List<Path>,
+  val outputs: List<Path>,
+  val classpath: List<Path>,
+  val bootClasspath: List<Path>,
+  val gen: Path,
+  val projectName: String,
+  val errorReporter: ErrorReporter = ErrorReporterImpl()
+) {
+
+  constructor(
```

Comment:
I think we can drop this one. Anyway its ABI isn't backward compatible.

Reply:
done